lua/dapui/render/loop.lua

```diff
@@ -54,9 +54,22 @@ function M.register_buffer(element_name, buf)
       .. buf
       .. ")"
   )
+  vim.cmd("augroup DAPUIExpandLongLines" .. buf)
+  vim.cmd(
+    "autocmd CursorHold <buffer="
```

Comment:
It would be good to select buffers to enable such autocommands. In `config/init.lua` it would be
```lua
    elements = {
      -- Provide IDs as strings or tables with "id" and "size" keys
      {
        id = M.elements.SCOPES,
        size = 0.25, -- Can be float or integer > 1
        expand_long_lines = true,
      },
      { id = M.elements.BREAKPOINTS, size = 0.25 },
      { id = M.elements.STACKS, size = 0.25, expand_long_lines = true },
      { id = M.elements.WATCHES, size = 0.25 },
    },
```
to override by the user. Do you see any good alternative other than passing such choice as an additional argument of `M.register_buffer(element_name, buf)`?

Reply:
I wouldn't do this as part of the `register_buffer` the settings are specific to the element so let's pass in a `settings` table to `register_element`

Reply:
```lua
---@param element Element
---@param settings? table
function M.register_element(element, settings)
  canvas_states[element.name] = {
    element = element,
    buffers = {},
    listeners = { [M.EVENTS.RENDER] = {}, [M.EVENTS.CLOSE] = {} },
    settings = settings or {},
  }
end
```

This can then instruct the canvas in `M.run` to enable hover lines

Reply:
Please check if latest changes are OK.